### DISCUSSION TITLE
NDI from Win 11 PC to Macbook Air M1 looks blurry

### DISCUSSION THREAD

**Discussion by ThankyouOBS-NDI** [DistroAV/DistroAV#824] [Q&A - Support]
I have my windows PC running 29.0 with latest NDI and same goes for the mac. I'm connected by a gigabit switch via CAT5e ethernet and run the NDI stream from my Windows PC at 1080P to my Mac which has its base res and canvas set to 1080p. The problem is the recordings are blurry on my Mac even though I have my recording set to X264 at 50,000kb/s. If I record on my Windows machine directly to OBS with the same preset there is no blurriness. 

I feel like the mac is scaling it weirdly but I'm not sure how. Any advice appreciated!

Screen shot of two different streams - top one is Win PC, bottom is Mac

![NDI issue](https://user-images.githubusercontent.com/122703066/212670337-142a3f9f-d420-4e17-abaa-555638e99235.png)

Recent log from Mac - 

[2023-01-16 11-27-54.txt](https://github.com/Palakis/obs-ndi/files/10425425/2023-01-16.11-27-54.txt)

**Comment by ThankyouOBS-NDI** [comment]
I wanted to post a small update to see if I can get any insight as to why it's happening. 

I installed NDI Tools on my Windows PC and ran the Screen Capture app and then recorded that feed on OBS on my Mac - this fixed the blurriness and the stream now looks very good. Would that indicate there is something odd going on with the OBS plug in?

**Comment by paulpv** [comment]
NDI is not lossless, and it is possible NDI's own Screen Capture app might have access to higher quality capturing than obs-ndi's open source does.

View your OBS log (in Help menu) for "obs-ndi" and what version of NDI SDK it is using.
